### ISSUE TITLE
Hyprland - Failed to get clients

### ISSUE THREAD

**Issue by logno-dev** [pierrechevalier83/workstyle#65]
Workstyle seems to no longer work with hyprland. I'm using Arch linux with hyprland-git verson 0.30.0 and workstyle version 0.8.2 and I get the error:

`Failed to get clients: invalid type: string "0x0", expected struct Client at line 24 column 23`

This likely isn't enough info to resolve the issue, I'm not terribly experienced with reporting issues. I'll provide whatever further information you'd like.

**Comment by LDAP** [comment]
Should work with latest main (which is not yet released).

**Comment by lbunch1** [comment]
I'm not sure if this is still not released, but the issue hasn't resolved on my end. I did try to build from source from main about a week ago and the issue persisted. Perhaps my issue is not resolved in the latest commit as anticipated.

**Comment by pierrechevalier83** [comment]
It's still not released. I tried to release 0.9 today, but I noticed that I was blocked by the fact that I temporarily forked swayipc until a PR I made upstream gets merged: https://github.com/JayceFayne/swayipc-rs/pull/47. I noticed I was the blocking element there, as I hadn't addressed some review comments, so did it now. Once that gets upstream, I'll switch back to upstream and release a new version.

In the meantime, I haven't been able to reproduce the problem on top of main, and I'm also running Arch and Hyprland.
